prow/scripts/cluster-integration/control-plane-gke-integration.sh

```diff
@@ -365,6 +401,7 @@ function installControlPlane() {
   kubectl create namespace "kcp-installer"
   applyCommonOverrides "kcp-installer"
   applyControlPlaneOverrides
+  applyKebResources
```

Comment:
Please use this function inside applyControlPlaneOverrides, because it is basically related to KCP.